oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/util/FacetHelper.java

```diff
@@ -62,6 +62,14 @@ public static FacetsConfig getFacetsConfig(NodeBuilder definition) {
 
     public static Facets getFacets(IndexSearcher searcher, Query query, QueryIndex.IndexPlan plan,
                                    SecureFacetConfiguration secureFacetConfiguration) throws IOException {
+        // This method is just a fail safe in case it is being called via the old signature in any code that consumes oak.
```

Comment:
I don't think we need to worry about these - this class should not be exposed outside oak-lucene and it should be ok to change method signature any way we want.